### PR TITLE
[Javascript] add uwebsockets.js

### PR DESCRIPTION
**Pull request by aadityataparia**
closes https://github.com/the-benchmarker/web-frameworks/issues/1064

Before submitting your PR, please review the following checklist :

## If you are adding a framework

+ [x] A `Dockerfile` exists, or can be built ?
+ [x] I've updated `CI` config file (and **commited** `.travis.yml`)
~~~sh
bin/make ci_config
~~~
+ [x] I've updated `.dependabot` config file  (and **commited** `.dependabot/config.yml`)
~~~sh
bin/make deps_config
~~~
+ [x] All tests passes ?
~~~
export FRAMEWORK=<MY_FRAMEWORK>
shards install
shards build --release
bin/make config
bin/neph ${FRAMEWORK}
crystal spec
~~~

### CODE REVIEW

**Comment by dalisoft** [comment]
@aadityataparia I think uWebSocket.js will lead on list

**Comment by waghanza** [comment]
@dalisoft Remember that results are not stable, probably `uwebsocket` will be on top but will decrease after ..

**Comment by dalisoft** [comment]
@waghanza I know, but uWebSockets.js will be lead a lot of time until someone really creates really well optimized code as uWebSockets.js has more than 7K lines-of-code in C++ language which very hard to beat (it's my subjective theory, I am not expert, just a fan of performance)

**Comment by waghanza** [comment]
@dalisoft You seems in love with `uWebSocket` :stuck_out_tongue: 

`C++` is not the **heaven** language (some could perform better depending on who it's code)

File: node/uwebsockets.js/config.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+framework:
+  website: github.com/uNetworking/uWebSockets.js
```
Comment (waghanza) [inline node/uwebsockets.js/config.yaml]: use `github: uNetworking/uWebSockets.js` instead

**Comment by aadityataparia** [comment]
If sifrr is at top, then probably `uwebsockets.js` will lead for now. 😂

**Comment by waghanza** [comment]
@alexhultman did no consent to add `uWebSocket` on this list, https://github.com/the-benchmarker/web-frameworks/issues/1064

then I'll close this `PR`

**Pull request closed without merging**